Meden-Agan-27/travel-worms
# Reviews branch

What I did:
Created feature to see reviews listed in book show page
Create feature to add a new review

Files I edited:
routes.rb (nested reviews in books)
reviews controller (index, new, and create for now - edit/destroy later)
book -> show.html.erb 	(added simple form and reviews list)

How to test:
localhost:3000 -> See all books -> Click on one book -> Add a review -> See the review listed below

NOTE: I added some "import" lines to javascript files in order to make the radio buttons appear as stars. Yuval said this is not a priority, so I will finish it later if we decide to implement it.
